entrypoints/content/components/secondary/comment/Comment.tsx

```diff
@@ -0,0 +1,50 @@
+
+
+// Components:
+import { Avatar, AvatarFallback, AvatarImage } from '@/entrypoints/content/components/ui/avatar'
+import CommentAction from './CommentAction'
+
+// Types and Utils:
+import {  Comment } from 'types/comments-and-replies'
+import { getPostedTimeDistanceFromNow } from '@/entrypoints/content/utils/timeHelpers'
+
+interface CommentProps {
+    userInfo: {
```

Comment:
`import { RealtimeDatabaseUser } from 'types/realtime.database.ts'`

`user: RealtimeDatabaseUser`

user's profile picture will always be accessible via their UID, and always be a .png
this logic is not in place at the moment, so `profileImgUrl` is OK for now

Reply:
Currently hardcoding the profile url but we can fix that when we integrate the backend

Reply:
yo, just integrated it
check `entrypoints/content/utils/getPhotoURLFromUID.tsx`

UID -> URL